ironfish/src/rpc/routes/wallet/importSpendAccount.ts

```diff
@@ -5,20 +5,21 @@ import { generateKeyFromPrivateKey } from '@ironfish/rust-nodejs'
 import { v4 as uuid } from 'uuid'
 import * as yup from 'yup'
 import { ApiNamespace, router } from '../router'
```

Comment:
I don't think this is an issue since we are still in testnet, but this is a breaking change to the SDK. In the future, we would keep the route for `importAccount` and just redirect to `importSpendAccount`